netbout-web/src/main/java/com/netbout/rest/RsPage.java

```diff
@@ -54,6 +54,7 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @EqualsAndHashCode(callSuper = true)
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@amihaiemil can you remove all suppressions for `@checkstyle MultipleStringLiteralsCheck`? They are not needed anymore